Patch-Source: https://github.com/php/php-src/commit/809176dab0b617934e7ec4394bf00309a2ee9754
From 809176dab0b617934e7ec4394bf00309a2ee9754 Mon Sep 17 00:00:00 2001
From: "Christoph M. Becker" <cmbecker69@gmx.de>
Date: Tue, 27 Sep 2022 17:52:01 +0200
Subject: [PATCH] Fix new bug81726.phpt for PHP 8.0

The error message has slightly changed, so we adapt our expectations.
---
 ext/phar/tests/bug81726.phpt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/phar/tests/bug81726.phpt b/ext/phar/tests/bug81726.phpt
index 11a148c28ea9..b698f0803706 100644
--- a/ext/phar/tests/bug81726.phpt
+++ b/ext/phar/tests/bug81726.phpt
@@ -10,5 +10,5 @@ if (!extension_loaded("zlib")) die("skip zlib extension not available");
 var_dump(fopen("phar://" . __DIR__ . "/bug81726.gz", "r"));
 ?>
 --EXPECTF--
-Warning: fopen(phar://%s): failed to open stream: unable to decompress gzipped phar archive "%s" in %s on line %d
+Warning: fopen(phar://%s): Failed to open stream: unable to decompress gzipped phar archive "%s" in %s on line %d
 bool(false)
